cfme/infrastructure/datastore.py

```diff
@@ -2,14 +2,15 @@
 """
 import attr
 from lxml.html import document_fromstring
-from navmazing import NavigateToAttribute
+from navmazing import NavigateToAttribute, NavigateToSibling
 from widgetastic.exceptions import NoSuchElementException
 from widgetastic.utils import Version, VersionPick
 from widgetastic.widget import ParametrizedView, View, Text
 from widgetastic_patternfly import Dropdown, Accordion
```

Comment:
Required : Sort imports

Reply:
@sshveta it does not come under the scope of this PR, but its ok we have to sort those.....

Reply:
@digitronik : Yeah I know Nikhil but why not improve code while you are at it :)